README.md

```diff
@@ -47,30 +49,106 @@ Requirements
-`/usr/lib/archivematica/automation-tools`
-* Create virtualenv `/usr/share/python/automation-tools` and pip install
-requirements there
-* Create directories `/var/log/archivematica/automation-tools` and
-`/var/archivematica/automation-tools` owned by user `archivematica`, for
-log/database/PID files.
-* Create directory `/etc/archivematica/automation-tools` and add configuration
-files there. Files in the `etc/` directory of this repository can be used as an
-example (also see below for more about configuration)
+Follow each of the steps below to install automation-tools:
+
+1. Make the following directories
+
+```bash
+mkdir -p /usr/lib/archivematica/automation-tools &&
+mkdir /usr/share/python/automation-tools &&
+mkdir -p /var/log/archivematica/automation-tools &&
+mkdir /var/archivematica/automation-tools &&
+mkdir /etc/archivematica/automation-tools
+```
+
+1. Change ownership of the directories to be owned by archivematica (group
+archivematica)
+
+```bash
+chown archivematica:archivematica /var/log/archivematica/automation-tools &&
+chown archivematica:archivematica /var/archivematica/automation-tools
+```
+
+1. Clone this automation-tools repository into your `usr/lib`
+
+```bash
+cd /usr/lib/archivematica/automation-tools/
+git clone https://github.com/artefactual/automation-tools.git .
+```
+
+1. Set up Python's virtual environment
+
+```bash
+cd /usr/share/python/automation-tools
+virtualenv venv
+source venv/bin/activate
+pip install -r /usr/lib/archivematica/automation-tools/requirements.txt
+```
+
+1. (Optional) Update and retrieve required packages
+
+```bash
+apt-get update
+apt-get install p7zip-full
+```
+
+You may also want/need to `apt-get install sudo && apt-get install vim`.
+
+To use automation-tools, `cp` the scripts you want to use into your
+`/etc/archivematica/automation-tools` directory.
+
+Follow the steps below as an example for setting up a script.
+
+1. Copy the configuration file `cp
+   /usr/lib/archivematica/automation-tools/etc/transfers.conf
+   /etc/archivematica/automation-tools/`
+
+1. Copy the bash script
+
+`cp /usr/lib/archivematica/automation-tools/etc/transfer-script.sh
+/etc/archivematica/automation-tools/`
+
+1. Create a transfer source for the automation-tools in the storage service
+
+You can use an existing transfer source, or set up a new one.
+
+1. Your default bash script should be configured along the lines of this:
+
+`transfer-script.sh`:
+
+```bash
+#!/bin/bash
+# docker transfer script example
+# /etc/archivematica/automation-tools/transfer-script.sh
+cd /usr/lib/archivematica/automation-tools/
+python -m transfers.transfer \
+ --am-url http://archivematica-dashboard:8000 \
+ --ss-url http://archivematica-storage-service:8000 \
+ --user test \
+ --api-key test  \
+ --ss-user test \
+ --ss-api-key test \
+ --transfer-source <transfer-source-in-storage-service> \
+ --config-file transfers.conf
+```
+
+5. Run the shell script!
+
+`$ ./transfer-script.sh` and you should have success!
```

Comment:
I think we're missing a step {n} to `chmod +x` the script to add executable permissions before running.